lisp/ess-s-lang.el

```diff
@@ -747,7 +747,6 @@ underscore character."
   (interactive)
   (require 'ess-mode); typically unnecessary
   (require 'ess-inf); dito
-  (define-key ess-mode-map          "\C-cf" 'ess-insert-function-outline)
```

Comment:
Sorry, but why is this one deleted? Whoever calls `ess-add-MM-keys` expects that key to be defined. I think it should be left alone. Plus the next line binds `C-c w` which you left as it is.

Reply:
Because that function doesn't exist anymore (unless users load old/ or add it manually).

Reply:
That's weird :/ Then we should surely delete the key, indeed.